ci-operator/step-registry/openstack/provision/etcd-disk-speed/openstack-provision-etcd-disk-speed-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
This step could be shared with other platforms to avoid duplication in the future, but this seems to be a common pattern in this repo.